### ISSUE TITLE
This repo is now obsolete

### ISSUE THREAD

**Issue by ksclarke** [webjars/openseadragon#6]
With the latest OpenSeadragon release the library is now available through Bower and/or NPM. I'd originally created this manual packaging for OpenSeadragon, but it is possible to now just use the Bower source for it. I've just triggered the deploy on the WebJars site using the Bower source and it seems to have worked fine.

Should this repo be deleted or a deprecation note be put in the README?

**Comment by jamesward** [comment]
Thanks! We usually keep these around in case someone still wants to use the Classic WebJar.

**Comment by ksclarke** [comment]
Hmm, maybe the process didn't work. I selected the "Deploy from Bower" option yesterday before I opened this ticket and it told me the deploy had worked but that it would be awhile before it would show up in Maven (there should be a 2.3.0 version there).

Today, I still don't see it: http://central.maven.org/maven2/org/webjars/openseadragon/

Are there issues with switching from a manually deployed project to one that uses Bower as a source?
